packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -930,8 +930,7 @@ class RawGestureDetectorState extends State<RawGestureDetector> {
     }
   }
 
-  /// This method can be called outside of the build phase to filter the list of
-  /// available semantic actions.
+  /// This method can be called to filter the list of available semantic actions.
```

Comment:
This should document when it is valid to call this method (e.g. what the assert is asserting).